<launch>
<node pkg="amcl" type="amcl" name="amcl" args="scan:=base_scan"/> 
<param name="min_dist_d" value="0.03" />


<node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
	<rosparam file="/home/lulu/myProjects/University/Year4/Robotics/Project/src/snake/scripts/costmap_common_params.yaml" command="load" ns="global_costmap" /> 
    <rosparam file="/home/lulu/myProjects/University/Year4/Robotics/Project/src/snake/scripts/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="/home/lulu/myProjects/University/Year4/Robotics/Project/src/snake/scripts/global_costmap_params.yaml" command="load" /> 
    <rosparam file="/home/lulu/myProjects/University/Year4/Robotics/Project/src/snake/scripts/local_costmap_params.yaml" command="load" /> 
    <rosparam file="/home/lulu/myProjects/University/Year4/Robotics/Project/src/snake/scripts/base_local_planner_params.yaml" command="load" />
    <!-- <rosparam file="/home/lulu/myProjects/University/Year4/Robotics/Project/src/snake/scripts/move_base_params.yaml" command="load" /> -->
</node>

</launch>
